src/drm.h

```diff
@@ -92,15 +92,15 @@ class Property : public Object {
   bool isImmutable() const { return flags_ & DRM_MODE_PROP_IMMUTABLE; }
 
   std::vector<uint64_t> values() const { return values_; }
-  const std::map<uint32_t, std::string>& enums() const { return enums_; }
+  const std::map<size_t, std::string>& enums() const { return enums_; }
   std::vector<uint32_t> blobs() const { return blobs_; }
 
  private:
```

Comment:
What you should do here alternatively:

![image](https://user-images.githubusercontent.com/1694275/164473199-57be9718-02a3-4de5-bd7e-f805332e3953.png)

If you do:

```
$ grep -r -A 1 "struct drm_mode_property_enum {" /usr/include/
/usr/include/drm/drm_mode.h:struct drm_mode_property_enum {
/usr/include/drm/drm_mode.h-	__u64 value;
--
/usr/include/libdrm/drm_mode.h:struct drm_mode_property_enum {
/usr/include/libdrm/drm_mode.h-	__u64 value;
```
You will see this

property->enums[i].value

is of type __u64

so these should be __u64 rather than size_t.

Actually your changes in camera_session.cpp more correctly should have been size_t because that's what ".size()" returns, if you want to fix.